gui/widget.cpp

```diff
@@ -703,9 +708,13 @@ void CheckboxWidget::setState(bool state) {
 	}
 	sendCommand(_cmd, _state);
 }
+	
+void CheckboxWidget::setOverride(bool override) {
+		_override = override; 
```

Comment:
Wrong indentation, another instance of bad variable name. `override` is a keyword